Automattic/wp-calypso
# Help Center: send missing url

Fixes https://github.com/Automattic/dotcom-forge/issues/9774

## Proposed Changes

* We weren't sending the customer site in the user fields before creating a conversation.

## Why are these changes being made?


* The site url should be passed when submitting the user fields

## Testing Instructions



* Build this PR
* Use the `help-center-experience` feature flag
* Create a new conversation
* Ensure that the url is passed when submitting the user fields
* 
<img width="820" alt="Screenshot 2024-11-12 at 16 53 07" src="https://github.com/user-attachments/assets/53dfa972-42f9-449f-8e33-ed8691e0e729">
